CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## v0.7.2
+
+### Fix
+  - Add an ability to receive GET and POST requests.
+  
```

Comment:
Add `$this->identityProvider->getSsoBinding()` is deprecated and no more used

Reply:
But we use it in the `src/Resources/views/Metadata/metadata.xml.twig`
Should I remove it?

Reply:
You're right ! 

The issue is that you cannot support both binding in the same route I think considering the metadata example here: https://en.wikipedia.org/wiki/SAML_2.0#SAML_2.0_Metadata